<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatWindow</class>
 <widget class="QWidget" name="GroupChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #C6C1BD</string>
  </property>
  <widget class="QTextEdit" name="GCChatDisplay">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>60</y>
     <width>391</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#EFEEEE;
color: #26313E; 

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inviteButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9C857B</string>
   </property>
   <property name="text">
    <string>초대</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GCMessageInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#EFEEEE;
color: #26313E; 

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GCSendButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>640</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9C857B</string>
   </property>
   <property name="text">
    <string>전송</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentRoomLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>16</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>궁서</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LANssenger</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
